README.rst

```diff
@@ -1013,6 +1013,17 @@ Choose from the list of available rules:
 
   The correct case must be used for standard PHP types in phpdoc.
 
+* **phpdoc_types_order**
+
+  Sorts PHPDoc types.
+
+  Configuration options:
+
+  - ``null_position`` (``'always_first'``, ``'always_last'``, ``'default'``): forces the
+    position of ``null``; defaults to ``'always_first'``
+  - ``sort_algorithm`` (``'alpha'``, ``'none'``): the sorting algorithm to apply;
+    defaults to ``'none'``
```

Comment:
I'm not sure most people would use `alpha`, so it wouldn't be a sane default.

I'm not a big fan of alphabetical sorting TBH, I would rather use something like https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2825#discussion_r124588919.